packages/website/docs/using-query-suggestions-plugin.md

```diff
@@ -2,7 +2,137 @@
+
+## Getting started
+
+First, begin with some boilerplate for the autocomplete implementation—create a file called `index.js` in your `src` directory, and add the boilerplate below:
+
+```js
+import { autocomplete } from '@algolia/autocomplete-js';
+
+// Instantiate the autocomplete instance
+autocomplete({
+  container: '#autocomplete',
+  plugins: [],
+  openOnFocus: true,
+});
+```
+
+This boilerplate assumes you want to insert the autocomplete into a DOM element with `autocomplete` as an [id](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id). You should change the [`container`](autocomplete-js/#container) to [match your markup](basic-options). Setting [`openOnFocus`](autocomplete-js/#openonfocus) to `true` ensures that the dropdown appears as soon as a user clicks on the input.
+
+For now, `plugins` is an empty array, but you'll learn how to add the Query Suggestions plugin next.
+
+## Adding Query Suggestions
+
+The [`autocomplete-plugin-query-suggestions`](createQuerySuggestionsPlugin) package provides the [`createQuerySuggestionsPlugin`](createQuerySuggestionsPlugin) function for creating a Query Suggestions plugin out-of-the-box.
+
+It requires an [Algolia search client](https://www.algolia.com/doc/api-client/getting-started/install/javascript/) initialized with an [Algolia application ID and API key](https://www.algolia.com/doc/guides/sending-and-managing-data/send-and-update-your-data/how-to/importing-with-the-api/#application-id) and an `indexName`. The `indexName` is the name of your [Query Suggestions](https://www.algolia.com/doc/guides/building-search-ui/ui-and-ux-patterns/query-suggestions/js/) index. **You must have a populated [Query Suggestions](https://www.algolia.com/doc/guides/building-search-ui/ui-and-ux-patterns/query-suggestions/js/#implementing-query-suggestions) index to use this plugin.**
+
+```diff
+import { autocomplete } from '@algolia/autocomplete-js';
++ import { createQuerySuggestionsPlugin } from '@algolia/autocomplete-plugin-query-suggestions';
++
++ const searchClient = algoliasearch(
++   'yourAppID',
++   'yourSearchApiKey'
++ );
++
++ const querySuggestionsPlugin = createQuerySuggestionsPlugin({
++   searchClient,
++   indexName: 'yourQuerySuggestionsIndexName',
++   },
++ });
++
+autocomplete({
+  container: '#autocomplete',
+-  plugins: [],
++  plugins: [querySuggestionsPlugin],
+  openOnFocus: true,
+});
+```
+
+You can optionally pass a `getSearchParams` function to apply [Algolia query parameters](https://www.algolia.com/doc/api-reference/api-parameters/) to the suggestions returned from the plugin. This is particularly useful if you need to align your Query Suggestions with other sections displayed in the autocomplete, like [recent searches](using-recent-searches-plugin).
```

Comment:
Is it worth mentioning this here, or should it only go in the "multi-source" guide?